### PR TITLE
Add payload radio

### PR DESCRIPTION
**Pull request by bbrk24**
The code is mostly untested, but I did make sure the number format works:

|  Number   |   Output    |
|-----------|-------------|
| 0.0       | `+00000.00` |
| -1.0      | `-00001.00` |
| 1.2345    | `+00001.23` |
| 11111.111 | `+11111.11` |

As mentioned on Discord, the DPST switch is needed to be able to upload new code to the Arduino, and isn't strictly necessary. I don't anticipate needing level shifters but testing is pending.

### CODE REVIEW

**Review by CClarke513 — APPROVED** [review]
LGTM

File: payload_bay_micro.ino [hunk 1]
```diff
@@ -4,9 +4,28 @@
...
 #include "Buffer.h"
 
+// How often to signal the radio, in Hertz
+const unsigned long RADIO_FREQ = 20;
+// The last time the altitude was transmitted, in us
+unsigned long lastRadioTime = 0;
+
 Altimeter alt;
 
+void sendToRadio(float altitude) {
```
Comment (DarthSethious1) [inline payload_bay_micro.ino:14]: Do we have a packet design for the radio bay as well now? I would like that to be put in a table somewhere (even discord would be fine)

File: payload_bay_micro.ino [hunk 2]
```diff
@@ -30,7 +62,7 @@ void loop() {
   switch (mode) {
   case BELOW_5K:
```
Comment (DarthSethious1) [inline payload_bay_micro.ino:65]: why are we not adding a point to the buffer to be sent to the radio in the "BELOW_5K" mode but we are in the other two? Wouldn't we want ot add a poin to the buffer to be sent to the radio in all modes?

Comment (bbrk24) [reply payload_bay_micro.ino:65]: you're right, that's an oversight on my part.

**Review by DarthSethious1 — APPROVED** [review]

**Pull request merged**